docs/versioned-plugins/outputs/elasticsearch-v11.22.10.asciidoc

```diff
@@ -0,0 +1,1422 @@
+:plugin: elasticsearch
+:type: output
+:no_codec:
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v11.22.10
+:release_date: 2024-12-09
+:changelog_url: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/v11.22.10/CHANGELOG.md
```

Comment:
@karenzone Note to self: Changelog entry links to wrong PR.  
Related: https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1196